src/DGtal/io/Display3D.h

```diff
@@ -854,9 +836,6 @@ namespace DGtal
     ///
     std::vector< QuadD3D > myPrismList;
 
```

Comment:
This is related to another PR that fixes Board3D quad export and which is still pending. Without these commits I cannot check the ticked colormap on surfaces